### ISSUE TITLE
Update pages.ko progress states in https://lukwebsforge.github.io/tldri18n/

### ISSUE THREAD

**Issue by IMHOJEONG** [LukWebsForge/TldrProgress#38]
hello. My name is IMHOJEONG, which is translating ko at https://lukwebsforge.github.io/tldri18n/. Although it is currently translated, an issue has been created because it appears as untranslated or links to the wrong document github address.

Below is a screen capture for reference.

<img width="1389" alt="Screenshot 2023-11-24 at 4 51 47 PM" src="https://github.com/LukWebsForge/TldrProgress/assets/11308147/a838d57f-279c-48dd-9a45-2140014ac454">

**Comment by LukBukkit** [comment]
Hi, thanks for the bug report. I've confirmed the issue and I'll look into it as soon as possible. However, currently I have to finish a few projects for university and thus it could take some time until the issues get fixed.

**Comment by LukBukkit** [comment]
Luckily, I could identify and fix the two issues very quickly:
- The translation progress wasn't updated because the local copy of the tldr repository used to generate the website on my server had some issues. I've recreated the repository, and now the page generation works again. You might have to do a force-reload on our browser to see the latest version of the data.
- GitHub apparently changed the URL scheme for creating new documents in a repository. I've adjusted the code for generating the link. This fix will be live after midnight (UTC).
